src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random randomNumber = new Random();
+
+    private final ColorSupplier color = new ColorSupplier();
+
+    private final Ball ball = new Ball();

```

Comment:
```suggestion
```
create an instance inside the method, because you should create different objects each time this method is invoked
With this implementation, you return the reference to this ball and just change it each time - which can cause some problems. You a re only changing the ball